src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,18 @@
+
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private final Color[] colors;
+    private final Random random;
+
+    public ColorSupplier(Color[] colors) {
+        this.colors = colors;
+        this.random = new Random();
+    }
+
+    public Color getRandomColor() {
+        return colors[random.nextInt(colors.length)];
```

Comment:
Let's keep all necessary logic inside this class, so delete this constructor and initialize random and Color[] inside this class.

Reply:
i don't understand what i must to do here, because my program working

Reply:
Working program is one thing, your code should also follow some rules and be readable.